mix.exs

```diff
@@ -30,7 +30,7 @@ defmodule BambooSmtp.Mixfile do
       {:earmark, ">= 1.0.3", only: :dev},
       {:ex_doc, "~> 0.14.5", only: :dev},
       {:excoveralls, "~> 0.6.0", only: :test},
-      {:gen_smtp, "~> 0.11.0"},
+      {:gen_smtp, github: "Vagabond/gen_smtp", branch: "master"},
```

Comment:
To be honest, I'm a bit worried to directly use the `master` branch of `gen_smtp`. 

Do you know why they don't release a new version?

At least, we could specify the specific SHA we want to use. So, if they break something, or change their API, that can't break Bamboo SMTP.